test/e2e/support/utils.js

```diff
@@ -86,31 +86,22 @@ export async function visitAdmin( adminPath, query ) {
 	}
 }
 
-export async function newPost( postType, disableTips = true ) {
+export async function newPost( { postType, enableTips = false } = {} ) {
 	await visitAdmin( 'post-new.php', postType ? 'post_type=' + postType : '' );
 
-	if ( disableTips ) {
+	const tipsEnabled = await page.evaluate( () => wp.data.select( 'core/nux' ).areTipsEnabled() );
+	if ( tipsEnabled && ! enableTips ) {
```

Comment:
It feels like we should be able to express this function more simply. Why do we need to check `areTipsEnabled` at all? Shouldn't this be accounted for by the effect of the `dispatch` ?

```js
await page.evaluate( ( _enableTips ) => {
	const action = _enableTips ? 'enableTips' : 'disableTips';
	wp.data.dispatch( 'core/nux' )[ action ]();
}, enableTips );

if ( enableTips ) {
	await page.reload();
}
```

Reply:
Yeah nice idea 👍